deploy/kustomize/base/06_subscription.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: hco-operatorhub
+  namespace: ${TARGET_NAMESPACE}
+spec:
+  source: ${CSC_SOURCE}
+  sourceNamespace: ${MARKETPLACE_NAMESPACE}
+  name: kubevirt-hyperconverged
+  startingCSV: kubevirt-hyperconverged-operator.v${HCO_VERSION}
+  channel: "${HCO_CHANNEL}"
+  installPlanApproval: Automatic
```

Comment:
Can we optionally handle KVM_EMULATION variable as we are doing here: https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/deploy/deploy_imageregistry.sh#L80 ?

Reply:
I can add the `spec.config` block on `subsciption` manifest the same way using multiline-variable substitution, or by using the `patch` feature of kustomize that will be in another yaml. What to you think is the preferred way?

Reply:
Since the existence of `subscription.spec.config` section is determined by the env var KVM_EMULATION, I decided to add it using a variable substitution, to avoid creating a new overlay branch for this.